mealie/services/user_services/registration_service.py

```diff
@@ -54,6 +54,11 @@ def _register_new_group(self) -> GroupInDB:
     def register_user(self, registration: CreateUserRegistration) -> PrivateUser:
         self.registration = registration
 
+        if self.repos.users.get_by_username(registration.username):
+            raise HTTPException(status.HTTP_400_BAD_REQUEST, {"message": "A user with this username already exists"})
+        elif self.repos.users.get(registration.email, "email"):
+            raise HTTPException(status.HTTP_400_BAD_REQUEST, {"message": "A user with this email already exists"})
+
```

Comment:
> duplicate records should return the http error code 409. I could unfortunately not find where I could define it.

You should be able to use status.HTTP_409 instead here right?

Reply:
status.HTTP_409 is undefined. I understood from the fastapi docs that is was defined in the `response_model`

Reply:
It's status.HTTP_409_CONFLICT, you can define the return success message in the decorator, if you're raising an error you'll pass it into the HTTPException from the fastapi module.